notebooks/02_dj_webinar.ipynb

```diff
@@ -0,0 +1,1403 @@
+{
```

Comment:
<p>Given that it was sometime ago that the project description was given (with some probably encountering for the first time), I'd say it's a good idea to provide the project description at the very beginning of this notebook.</p><p> </p><p>Also let's stick for consistency in <code>NumPy</code>  and not <code>Numpy</code>.  Could have been me who introduced that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/datajoint-webinar/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I added the project description at the beginning.</p>